
ifmap is already defined in net/if.h - so disable most parts of the file for
user-space applications.

  - Rene Rebe <rene@rocklinux-consulting.de>

--- usr/include/linux/if.h.orig	2004-09-24 21:27:57.527619915 +0000
+++ usr/include/linux/if.h	2004-09-24 21:25:18.511007038 +0000
@@ -91,6 +91,8 @@
  *	being very small might be worth keeping for clean configuration.
  */
 
+#ifdef __KERNEL__
+
 struct ifmap 
 {
 	unsigned long mem_start;
@@ -189,4 +191,6 @@
 #define	ifc_buf	ifc_ifcu.ifcu_buf		/* buffer address	*/
 #define	ifc_req	ifc_ifcu.ifcu_req		/* array of structures	*/
 
+#endif /* __KERNEL__ */
+
 #endif /* _LINUX_IF_H */
